biome.json

```diff
@@ -0,0 +1 @@
+typescript/biome.json
```

Comment:
Is this necessary on this PR?

Reply:
I think we should include it, without it most peoples IDEs won't pick up on biome and they effectively won't have code formatting

Reply:
ok